ui/app/models/pki/crl.js

```diff
@@ -4,15 +4,84 @@
  */
 
 import Model, { attr } from '@ember-data/model';
+import { withFormFields } from 'vault/decorators/model-form-fields';
+import lazyCapabilities, { apiPath } from 'vault/macros/lazy-capabilities';
 
+@withFormFields(['crlExpiryData', 'autoRebuildData', 'deltaCrlBuildingData', 'ocspExpiryData'])
 export default class PkiCrlModel extends Model {
   // This model uses the backend value as the model ID
-  get useOpenAPI() {
-    return true;
-  }
 
+  // API params are updated in the serializer from the attr objects below
   @attr('string') expiry;
-  @attr('boolean') autoRebuild;
-  @attr('string') ocspExpiry;
+  @attr('boolean') disable;
   @attr('boolean') ocspDisable;
+  @attr('string') ocspExpiry;
+  @attr('boolean') autoRebuild;
+  @attr('string') autoRebuildGracePeriod;
+  @attr('boolean') enableDelta;
+  @attr('string') deltaRebuildInterval;
+
+  @attr('object', {
+    label: 'Auto-rebuild on',
+    labelDisabled: 'Auto-rebuild off',
+    editType: 'ttl',
+    defaultValue() {
+      return { enabled: false, duration: '12h' };
+    },
+    helperTextEnabled: 'Vault will rebuild the CRL in the below grace period before expiration',
```

Comment:
Probably a design question, but I'm seeing some helperText with periods and some without. I know in general we're a bit all over with this, but thought I would point it out.

Reply:
Just double checked and these are all copy and pasted from the designs - probably worth following up with the latest iteration of the design team and see how we want to handle!